frontend/package.json

```diff
@@ -44,7 +44,7 @@
     "@openshift/dynamic-plugin-sdk": "^3.0.0",
     "@openshift/dynamic-plugin-sdk-utils": "^1.2.0",
     "@patternfly/patternfly": "4.224.2",
-    "@patternfly/quickstarts": "^2.3.1",
+    "@patternfly/quickstarts": "^2.4.3",
```

Comment:
FWIW, we are at [5.1.0](https://github.com/opendatahub-io/odh-dashboard/blob/845160368eae5fff39fc282c36d99a195f24b7d7/frontend/package.json#L47)